store/demo/dev/10111__task_dag.sql

```diff
@@ -0,0 +1,29 @@
+-- task_dag for task dependency
+
+INSERT INTO
+    task_dag (
+        id,
+        from_task_id,
+        to_task_id
+    )
+VALUES
+    (
+        101,
```

Comment:
This should start with 11101.

The file name starts with 10111, 111xx the first one will be 11101 if we look at other examples in the directory.